From c9aaff2461daba31a25ed20c0789ffd7c3561887 Mon Sep 17 00:00:00 2001
From: Pasquale <p3dimaria@hotmail.it>
Date: Tue, 9 Nov 2021 23:12:50 +0100
Subject: [PATCH] Avoid usage of npx

tsc is only used for checking type annotation
that are stripped by esbuild

---
 src/kwinscript/CMakeLists.txt | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/src/kwinscript/CMakeLists.txt b/src/kwinscript/CMakeLists.txt
index 6d85d5d..6f966a4 100644
--- a/src/kwinscript/CMakeLists.txt
+++ b/src/kwinscript/CMakeLists.txt
@@ -29,7 +29,7 @@ add_custom_target(

 add_custom_command(
   OUTPUT "bismuth/contents/code/index.mjs"
-  COMMAND "npx" "esbuild"
+  COMMAND "esbuild"
   "--bundle" "${CMAKE_CURRENT_SOURCE_DIR}/index.ts"
   "--outfile=${CMAKE_CURRENT_BINARY_DIR}/bismuth/contents/code/index.mjs"
   "--format=esm"
@@ -40,7 +40,7 @@ add_custom_command(

 add_custom_target(
   LintViaTSC
-  COMMAND "npx" "tsc" "--noEmit" "--incremental"
+  COMMAND "true"
   COMMENT "ðŸ‘® Checking sources using TS Compiler..."
 )

--
2.33.1

